src/components/settings/releaseNotes/ReleaseNotesLayout.jsx

```diff
@@ -0,0 +1,77 @@
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { observer } from 'mobx-react';
+import { defineMessages, injectIntl } from 'react-intl';
+
+import { mdiClose } from '@mdi/js';
+import { Outlet } from 'react-router-dom';
+import ErrorBoundary from '../../util/ErrorBoundary';
+import Appear from '../../ui/effects/Appear';
+import Icon from '../../ui/icon';
+
+const messages = defineMessages({
+  closeSettings: {
+    id: 'settings.app.closeSettings',
+    defaultMessage: 'Close settings',
+  },
+});
+
+class ReleaseNotesLayout extends Component {
+  static propTypes = {
+    closeSettings: PropTypes.func.isRequired,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keydown', this.handleKeyDown.bind(this), false);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener(
+      'keydown',
+      // eslint-disable-next-line unicorn/no-invalid-remove-event-listener
+      this.handleKeyDown.bind(this),
+      false,
+    );
+  }
+
+  handleKeyDown(e) {
+    if (e.keyCode === 27) {
```

Comment:
please assign the `27` to a meaningful constant

Reply:
Also followed this: https://github.com/ferdium/ferdium-app/blob/b85efe342e727905eb227bf227a9d1f06ae48e79/src/components/settings/SettingsLayout.jsx#L38-L43

Should we assign a global constant for this?